hbase_storage/src/main/java/com/splicemachine/si/impl/server/AbstractSICompactionScanner.java

```diff
@@ -107,6 +109,10 @@ public boolean next(List<Cell> list) throws IOException {
         }
     }
 
+    public long getSize() {
```

Comment:
I suggest renaming size to numberOfScannedBytes or something like this. The size of a scanner is a bit vague.

Reply:
done.